pyphare/pyphare/pharesee/geometry.py

```diff
@@ -205,15 +238,23 @@ def particle_ghost_area_boxes(hierarchy):
     for ilvl, lvl in hierarchy.levels().items():
         for patch in lvl.patches:
 
-            patch_data = patch.patch_datas["particles"]
-            gbox = patch_data.ghost_box
-            box = patch.box
+            particles_ids = [key for key in patch.patch_datas.keys() if key.endswith("particles")]
```

Comment:
I think there's only actually one population in this case, just the name of it can vary, they just always end in "particles" but might have a different prefix